alvr/audio/src/lib.rs

```diff
@@ -57,7 +57,7 @@ pub struct AudioDevice {
 impl AudioDevice {
     pub fn new(
         linux_backend: Option<LinuxAudioBackend>,
-        id: AudioDeviceId,
+        id: &AudioDeviceId,
```

Comment:
Why this change? If needed you can clone `AudioDeviceId`.

Reply:
Nevermind, I think It's reasonable. `AudioDeviceId` may allocate a string and it seems the method does not need to consume the value so taking a reference is justified.